tests/output_guardrails/conftest.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+import pytest
+
+
+@pytest.fixture
+def mock_input_data(mock_project_root):
+    data = [
+        {
+            "question": "Question 1",
+            "expected_triggered": True,
+            "actual_triggered": True,
+            "expected_exact": "1, 5, 7",
+            "actual_exact": "1, 5, 7",
+        },
+        {
+            "question": "Question 2",
+            "expected_triggered": False,
+            "actual_triggered": True,
+            "expected_exact": "None",
+            "actual_exact": "1",
+        },
+    ]
+
+    path = mock_project_root / "input_data.jsonl"
+
+    with open(path, "w") as file:
+        for item in data:
+            json.dump(item, file)
+            file.write("\n")
+
+    return str(path)
```

Comment:
I might be wrong, but I don't think we need to convert this to a string from a path (I imagine it might be something to fix in existing code)